include/mbedtls/ecdsa.h

```diff
@@ -1,10 +1,15 @@
 /**
  * \file ecdsa.h
  *
- * \brief Elliptic curve DSA
+ * \brief Elliptic curve DSA.
+ *
+ * The Elliptic Curve Digital Signature Algorithm (ECSDA) is defined in 
+ * <em>RFC-4492: Elliptic Curve Cryptography (ECC) Cipher Suites for Transport 
+ * Layer Security (TLS)</em>.
+ *                  
  */
-/*
- *  Copyright (C) 2006-2015, ARM Limited, All Rights Reserved
+/* 
```

Comment:
Blocker: Please remove the trailing whitespace.

Reply:
done